packages/babel-preset-gatsby/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "babel-preset-gatsby",
+  "version": "2.5.0",
+  "author": "Philipp Spiess <hello@philippspiess.com>",
+  "devDependencies": {
```

Comment:
These should all be _dependencies_ right, i.e. they're required by this package?